packages/flutter_tools/lib/src/platform_plugins.dart

```diff
@@ -86,13 +95,17 @@ class AndroidPlugin extends PluginPlatform implements NativeOrDartPlugin {
   @override
   bool isNative() => pluginClass != null;
 
+  @override
+  bool isFfi() => ffiPlugin == true;
+
   static bool validate(YamlMap yaml) {
```

Comment:
Something that we'll have to be careful about here is that plugins using the new metadata will break the build for any version of Flutter older than this change. We'll probably need to make the template require a min version of Flutter equal to whatever this change lands in.

Reply:
This change wasn't made. The pubspec.yaml still unconditionally has:

```
flutter: ">=2.5.0"
```